Jenkinsfile

```diff
@@ -75,6 +75,26 @@ pipeline {
       }
     }
 
+    stage('Package Versions') {
+      parallel {
+        stage('CPU Diff') {
+          steps {
+            sh '''#!/bin/bash
+            ./diff
```

Comment:
You should pull the `${PRETEST_TAG}` and specify the target when calling diff. 

There is no guarantee that the same worker than the build step will run this code.

By default, 

Same applies to the `GPU Diff` stage.

See: https://github.com/Kaggle/docker-python/blob/master/Jenkinsfile#L100-L101